front/pages/poke/index.tsx

```diff
@@ -56,7 +58,13 @@ const renderWorkspaces = (title: string, workspaces: PokeWorkspaceType[]) => (
                   </PokeTableCell>
                 </PokeTableRow>
                 <PokeTableRow>
-                  <PokeTableCell>
+                  <PokeTableCell
+                    style={{
+                      overflow: "hidden",
+                      textOverflow: "ellipsis",
+                      maxWidth: "200px",
+                    }}
```

Comment:
We should use tailwind here 😛. If the component does not accept a `className` feel free to augment the component so we can customize it.

Reply:
sorry, will fix :)